client/my-sites/customer-home/cards/actions/quick-links/style.scss

```
@@ -89,6 +89,10 @@
 			fill: var( --color-neutral-80 );
 		}
 	}
+
+	&:last-of-type {
+		box-shadow: 0 -1px 0 0px var( --color-border-subtle );
```

I think that resetting [this `margin-bottom: 1px`](https://github.com/Automattic/wp-calypso/blob/43e4cb271e3437d94e4eb48c65510474b39b2624/packages/components/src/card/style.scss#L22) fixes both the 2px border and the white small border I noted while testing:

```suggestion
		margin-bottom: 0;
```

---

Nice catch - that works for me too! :)